code/modules/clothing/modular_armor/attachments/modules.dm

```diff
@@ -121,6 +121,15 @@
 	slot = ATTACHMENT_SLOT_HEAD_MODULE
 	variants_by_parent_type = list(/obj/item/clothing/head/modular/marine/m10x = "tyr_head_xn", /obj/item/clothing/head/modular/marine/m10x/leader = "tyr_head_xn")
 
+/obj/item/armor_module/module/hod_head
+	name = "Hod Helmet System"
+	desc = "Designed for mounting on a modular helmet. When attached, this system provides substantial resistance to most gunshot wounds by providng high internal padding within the helmets structure."
+	icon = 'icons/mob/modular/modular_armor_modules.dmi'
+	icon_state = "mod_ff"
```

Comment:
Make a new item sprite. mimir hat is basically just a cropped mimir suit, so you can do the same.